query/src/main/scala/filodb/query/exec/rangefn/RangeFunction.scala

```diff
@@ -496,6 +521,30 @@ class LastSampleChunkedFunctionD extends LastSampleChunkedFuncDblVal() {
   }
 }
 
+// Return the last sample within the window
+class LastSampleInWindowChunkedFunctionD extends LastSampleChunkedFunctionD {
```

Comment:
The abstract class and implementation for double and histogram would be useful. Lets quickly add them and move on.

Reply:
i have added the stack to support double, long and histo types